io/plugins/eu.esdihumboldt.hale.io.jdbc/src/eu/esdihumboldt/hale/io/jdbc/extension/DriverConfigurationExtension.java

```diff
@@ -66,4 +68,19 @@ protected DriverConfiguration create(String elementId, IConfigurationElement ele
 		return null;
 	}
 
+	/**
+	 * To find driver using driver configuration
+	 * 
+	 * @param jdbcUri uri of JDBC connection
+	 * @return {@link DriverConfiguration}
+	 */
+	public DriverConfiguration findDriver(URI jdbcUri) {
+		for (DriverConfiguration config : getElements()) {
+			if (config.matchURIPrefix(jdbcUri))
+				return config;
+		}
+
+		return null;
```

Comment:
This method may return null. You should document that, and where you use it also handle that case.